src/components/link/link.component.tsx

```diff
@@ -109,6 +112,7 @@ export const Link = React.forwardRef<
           type={icon}
           bgSize="extra-small"
           disabled={disabled}
+          hideAriaLabel={hideAriaLabel}
```

Comment:
**question**: I could well be missing something, but can't we just change the `ariaLabel={ariaLabel}` to `ariaLabel={hideAriaLabel ? undefined : ariaLabel}`, to avoid needing a `hideAriaLabel` prop at all on `Icon`? (You'd still need it on Link though, assuming you ever wanted a <span> with an incorrect aria-label for an icon inside a Link.)

Reply:
I'll message you privately about this as It'll need some further explaining.